tomviz/LoadDataReaction.cxx

```diff
@@ -115,6 +116,7 @@ QList<DataSource*> LoadDataReaction::loadData()
           << "JPeg Image files (*.jpg *.jpeg)"
           << "PNG Image files (*.png)"
           << "TIFF Image files (*.tiff *.tif)"
+          << "Data Exchange files (*.h5)"
```

Comment:
This is just the HDF5 generic extension, we probably want to avoid claiming it is just Data Exchange.